### ISSUE TITLE
Unhandled error, not going through correct error-handling?

### ISSUE THREAD

**Issue by GeoffreyPlitt** [balderdashy/skipper-s3#37]
Hi,

I'm using skipper-s3 0.5.6. I have error handling hooked up both with event-style ```.on('error', (err) => { ...}``` and callback-style ```function(err, result) {...}``` handlers. Yet we're still getting this error happening that isn't going through these handlers (see screenshot). I can't tell what the actual error reason/code is because I can't trap the error.

Please advise, thanks.
![image](https://cloud.githubusercontent.com/assets/382821/10828650/db5a493e-7e32-11e5-9f67-6a3c369410bb.png)

**Comment by Emiyashirou** [comment]
Hi, 
I'm using skipper-s3 0.5.6 and have abort request issue mentioned in [#49](https://github.com/balderdashy/skipper/issues/49) and [#74](https://github.com/balderdashy/skipper/pull/78/files)
Looks like it has been fixed in skipper and skipper-disk, but in s3 adapter, it crashed any way.
I tried to fix this by changing (line 210 in index.js):

**receiver__.once('error', function (err) {**
      **// console.log('ERROR ON RECEIVER__ ::',err);**
    **});**

to:

**receiver__.on('error', function (err) {**
      **// console.log('ERROR ON RECEIVER__ ::',err);**
    **});**

and it works. 
So it looks like same solution in [#74](https://github.com/balderdashy/skipper/pull/78/files) can be used here, can you fix this problem? Thanks.

**Comment by GeoffreyPlitt** [comment]
Cool. @sgress454 @TimofeyBiryukov @evert0n can we get this fixed?

**Comment by sailsbot** [comment]
Thanks for posting, @GeoffreyPlitt.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
